programs/bpf_loader/src/syscalls/mod.rs

```diff
@@ -4441,28 +4417,24 @@ mod tests {
             SBPFVersion::V3,
         )
         .unwrap();
-        let processed_sibling_instruction = translate_type_mut::<ProcessedSiblingInstruction>(
-            &memory_mapping,
-            VM_BASE_ADDRESS,
-            true,
-        )
-        .unwrap();
+        let processed_sibling_instruction =
+            unsafe { &mut *memory.as_mut_ptr().cast::<ProcessedSiblingInstruction>() };
```

Comment:
How is the translation not necessary here anymore?

Reply:
It is a test. It only uses that for the setup.

Reply:
Yep, but were are still converting the address from the memory mapping, aren't we?